documentation/tags.md

```diff
@@ -48,6 +48,7 @@ Catch
 Characters
 Chatbots
 Cryptocurrency
+C#
```

Comment:
I'd remove this as we generally don't have language topics as tags as they're redundant.

Reply:
Fixed